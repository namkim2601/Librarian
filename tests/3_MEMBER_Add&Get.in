MeMbeR nomembers
aDd MemBeR Nam Kim
add member Should Be 100001
member nosuchmember
member 100000
member 100001
exit
