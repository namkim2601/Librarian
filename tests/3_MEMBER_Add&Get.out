user: No members in system.

user: Success.

user: Success.

user: No such member in system.

user: 100000: Nam Kim

user: 100001: Should Be 100001

user: Ending Library process.
